accounts-db/src/read_only_accounts_cache.rs

```diff
@@ -104,15 +104,15 @@ impl ReadOnlyAccountsCache {
         let data_size = Arc::new(AtomicUsize::default());
         let stats = Arc::new(AtomicReadOnlyCacheStats::default());
         let (evict_sender, evict_receiver) = crossbeam_channel::bounded::<()>(1);
-        let evictor = Self::spawn_evictor(
-            evict_receiver,
-            max_data_size_lo,
-            max_data_size_hi,
-            data_size.clone(),
-            cache.clone(),
-            queue.clone(),
-            stats.clone(),
-        );
+        // let evictor = Self::spawn_evictor(
```

Comment:
Do you mind adding a code comment here for why we're removing this call? This will help other maintainers who lack context and also our future selves, thanks.